Gemfile

```diff
@@ -1,7 +1,7 @@
 source "https://rubygems.org"
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
-ruby "~> 3.2.0"
+ruby File.read(".ruby-version").strip
```

Comment:
Another opinionated improvement related to ruby version is that could be discussed is to read the version directly from `.ruby-version` on `Gemfile` so that we would need to change it only at one place.